src/events/crawler/_crawl.js

```diff
@@ -7,20 +10,25 @@ const cache = require('./cache')
  */
 module.exports = async function crawl (event) {
   try {
+    let { _useUTCdate } = event
+
     /**
      * Load the requested source
      */
     const source = getSource(event)
-    const { scrapers, _sourceKey } = source
+    const { _sourceKey } = source
 
     const timeLabel = `Crawl ${_sourceKey}`
     console.time(timeLabel)
 
     /**
      * Select the current scraper from the source's available scrapers
      */
-    // TODO actually calculate latest start date; this hack works for now
-    const scraper = scrapers[scrapers.length - 1]
```

Comment:
@ryanblock - I copied the corresponding section from _scrape.js, removed excess.  OK to you?